### PR TITLE
move the TSB templates to an install location

### PR DESCRIPTION
**Pull request by deads2k**
@ewolinetz @sdodson this separates the templates into
 1. apiserver config yaml for substitution by ansible
 2. rbac-template to be reconciled
 3. apiserver-template to accept the substituted apiserver config.yaml and be oc applied

@bparees @jim-minter This is splitting up pieces for ansible to consume

@stevekuznetsov Let's say I had this new requirement to synchronize the `install` folder from origin into a directory of @sdodson's choosing in openshift-ansible on every origin merge that changed a file in `install`?  This would let ownership of the file be clearly in origin by the developer working on it and @sdodson mentioned a way to actually use these origin files to drive the ansible install in CI, so that we get a matched set.

### CODE REVIEW

File: hack/update-generated-bindata.sh [hunk 1]
```diff
@@ -54,12 +54,12 @@ pushd "${OS_ROOT}" > /dev/null
...
     examples/jenkins/... \
     examples/quickstarts/cakephp-mysql.json \
-    examples/templateservicebroker/...
+    install/...
 
 popd > /dev/null
 
 # If you hit this, please reduce other tests instead of importing more
-if [[ "$( cat "${OUTPUT_PARENT}/test/extended/testdata/bindata.go" | wc -c )" -gt 920000 ]]; then
+if [[ "$( cat "${OUTPUT_PARENT}/test/extended/testdata/bindata.go" | wc -c )" -gt 1500000 ]]; then
```
Comment (bparees) [inline hack/update-generated-bindata.sh]: lol. don't tell clayton.

Comment (deads2k) [reply hack/update-generated-bindata.sh]: > lol. don't tell clayton.

problems I don't need to have.... I thought about making it a gig and walking away like it never happened :)

**Comment by bparees** [comment]
lgtm

File: hack/update-generated-bindata.sh (hunk 1 above)
Comment (stevekuznetsov) [reply hack/update-generated-bindata.sh]: hahaha

**Comment by ewolinetz** [comment]
@deads2k so when you say we reconcile the rbac file, is that a `oc apply -f` and then `oadm policy reconcile-cluster-roles`?

**Comment by deads2k** [comment]
> @deads2k so when you say we reconcile the rbac file, is that a oc apply -f and then oadm policy reconcile-cluster-roles?

`oc process -f rbac-template.yaml | oc auth reconcile`.  I've got to sort that command out with @fabianofranz .

@sjenning can you help find the missing pod logs.  It's the ones I asked about in #openshift-dev.  The kubelet seems to be saying that the pod was created, failed readiness probes, but all containers were removed before the CI could gather the logs.

**Pull request merged**